### PR TITLE
[ERE-209] Enable password validator on Angelman registry

### PR DESCRIPTION
**Pull request by alintreznai**

### CODE REVIEW

File: ang_scripts/force_password_change.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import django
+import logging
+
+logger = logging.getLogger(__name__)
+
+django.setup()
+
+
+def force_password_change():
+    from registry.groups.models import CustomUser
```
Comment (sztamas) [inline ang_scripts/force_password_change.py]: Imports at top of the file, please.

File: ang_scripts/force_password_change.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    logger.info("Forcing password change for all users")
+    count = CustomUser.objects.all().update(force_password_change=True)
```
Comment (sztamas) [inline ang_scripts/force_password_change.py]: Just as a note you can leave out the `all().`. ie `count = CustomUser.objects.update(force_password_change=True)`

**Review by sztamas — DISMISSED** [review]

**Review by sztamas — CHANGES_REQUESTED** [review]
Can you please move this script to TRRF scripts?
There is no need for it to be in Angelman, there is nothing Angelman-specific about it.

File: ang_scripts/force_password_change.py (hunk 1 above)
Comment (alintreznai) [reply ang_scripts/force_password_change.py]: Doesn't work in this case as this import needs to be done after django.setup() executes

**Comment by alintreznai** [comment]
Moved password change script to:https://github.com/eresearchqut/trrf/pull/259. Only kept the settings change here.

**Review by sztamas — APPROVED** [review]

**Pull request merged**